### PR TITLE
pretty print of vw audit

### PR DESCRIPTION
**Pull request by gupchup**
The main reason to do this was that I found we needed to use audit repeatedly to construct a map of the namespace:features -> hashes -> values. I found that printing it out in a tabular format helped in speeding up the process of debugging the model. Hopefully this benefits others too.

### CODE REVIEW

File: DataScience/vw_audit_pp.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    parse_audit_command(vw_cmd_output, audit_output_file)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('-m', '--model',        type=str, required=True)
+    parser.add_argument('-i', '--input',        type=str, required=True)
+    parser.add_argument('-o', '--output',       type=str, required=True)
+    parser.add_argument('-d', '--debug',        type=str, required=False, default=True)
```
Comment (gupchup) [inline DataScience/vw_audit_pp.py]: Should probably set default to False.

**Comment by JohnLangford** [comment]
Since this is VW-specific, it seems better to merge it into the VW repository, right?

**Comment by marco-rossi29** [comment]
Yes, that’s a good point. I’ll take a look at all VW specific code and move it to VW repo

Marco

**Comment by JohnLangford** [comment]
This seems like it could easily be in the utl directory of VW.

-John

**Pull request merged**